src/components/UsageBar.tsx

```diff
@@ -0,0 +1,93 @@
+'use client'
+
+import { ReactNode, useEffect, useState } from 'react'
+import { useKeyring, Plan } from '@w3ui/react-keyring'
+
+const B = 1024
+const MB = 1024 * B
+const GB = 1024 * MB
+const BarHeight = 10
+
+const PlanLimit: Record<string, number> = {
+  'did:web:free.web3.storage': 5 * GB,
+  'did:web:pro.web3.storage': 25 * GB,
+  'did:web:enterprise.web3.storage': 100 * GB
+}
+
+export function UsageBar (): ReactNode {
+  const [{ account }, { getPlan }] = useKeyring()
+  const [plan, setPlan] = useState<Plan>()
+  const [usage, setUsage] = useState<Record<string, number>>()
+
+  useEffect(() => {
+    if (!account) return
+    (async () => {
+      if (account) {
+        try {
+          const result = await getPlan(account as `${string}@${string}`)
+          if (result.error) throw new Error('getting plan', { cause: result.error })
+          setPlan(result.ok)
+        } catch (err) {
+          console.error(err)
+        }
+      }
+    })()
+  }, [account, getPlan])
+
+  useEffect(() => {
+    if (!account) return
+    // TODO: get usage by space
+  }, [account])
+
+  if (!plan) setPlan({ product: 'did:web:free.web3.storage' })
+  if (!usage) setUsage({
+    'did:key:z6MkjCoKJcunQgzihb4tXBYDd9xunhpGNoC14HEypgAe5cNW': 500_000_000,
+    'did:key:z6MketbAFtbeqDeVHxSzSSDH2PfMTquQ3vzZCcPFweXBGe3R': 200_000_000,
+    'did:key:z6MkoEBnTj96thGj7H7c1DrdNHF4AiA9VPDRVqTmp4teYd6B': 78_000_000,
+  })
```

Comment:
is this for demoing?

Reply:
Apolgies I should have put this as a draft PR.